### DISCUSSION TITLE
Manga Support

### DISCUSSION THREAD

**Discussion by amasterengineer** [advplyr/audiobookshelf#3493] [Q&A]
let me say this application is amazing and what i have been looking for for a long time. i do have a question about manga metadata support. does anyone know how to add a metadata provider like Jikan? this would allow me to have some sort of metadata for my manga collection without having to add it myself for each and everyone of the volumes.

**Comment by nichwall** [comment]
You can create a custom metadata provider to pull data from other sources, and then add the provider to ABS.

https://www.audiobookshelf.org/guides/custom-metadata-providers

**Reply by amasterengineer** [reply]
i have looked into it but i can not seem to understand how to apply this. i was hoping someone else have sorted it out to copy off of lol i am trying to sort out how to use and setup the API for mymangalist.net, but this is all new to me so im not making much progress
